packages/wayland/wayland.pacscript

```diff
@@ -0,0 +1,43 @@
+pkgname=wayland
+gives="wayland"
+pkgver=1.23.0
+pkgrel=1
+pkgdesc='A computer display server protocol'
+arch=('amd64' 'i386')
+url='https://wayland.freedesktop.org/'
+source=("https://gitlab.freedesktop.org/wayland/wayland/-/releases/${pkgver}/downloads/wayland-${pkgver}.tar.xz")
+sha256sums=('05b3e1574d3e67626b5974f862f36b5b427c7ceeb965cb36a4e6c2d342e45ab2')
+depends=('libglib2.0-0' 'libffi8' 'libexpat1' 'libxml2' 'libgnutls30')
+makedepends=('meson' 'cmake' 'pkg-config' 'libxslt1-dev' 'doxygen' 'xmlto' 'graphviz' 'docbook-xsl' 'libffi-dev' 'libffi8' 'libexpat-dev' 'libxml2-dev')
+provides=(libwayland-client0="${pkgver}"
+  libwayland-egl1="${pkgver}"
+  libwayland-server0="${pkgver}"
+  libwayland-cursor0="${pkgver}"
+  libwayland-dev="${pkgver}"
+  libwayland-egl-backend-dev="${pkgver}"
+  libwayland-doc="${pkgver}"
+  libwayland-bin="${pkgver}")
+replaces=(libwayland-client0 libwayland-egl1 libwayland-server0 libwayland-cursor0 libwayland-dev libwayland-egl-backend-dev libwayland-doc libwayland-bin)
+breaks=("${replaces[@]}")
+
+maintainer=("xdavius <xdavius@github.com>")
+
+build() {
+  cd "${pkgname}-${pkgver}"
+  meson --prefix=/usr build
+  meson compile -C build
+}
+
+check() {
+  cd "${pkgname}-${pkgver}"
+  meson test -C build --print-errorlogs
+}
+
+package() {
+  cd "${pkgname}-${pkgver}"
+  meson install -C build --destdir "${pkgdir}"
+  mkdir -p docs/share
+  mv "${pkgdir}"/usr/share/{doc,man} docs/share
```

Comment:
Why move those?

Reply:
It's a splited package in arch. I want it joined. I have certainly missed something.